framework/interface/api_handlers.py

```diff
@@ -166,6 +166,11 @@ def delete(self, session_id=None, action=None):
         try:
             self.get_component("session_db").delete_session(
                 int(session_id))
+            # gets the id of the default session. 
+            default_session = self.get_component("session_db").get_default_id()
```

Comment:
The default session's ID is after deletion because I noticed that when I try to delete the default session from the UI, it is deleted and a new "default session" is written into the database.